app/app/static_storage.py

```diff
@@ -47,6 +53,21 @@ def _save_content(self, obj, content, parameters):
         if not content_autoclose.closed:
             content_autoclose.close()
 
+    def save(self, name, content):
+        # record the clean file content (pre gzip)
+        file_content = content.file
+        # Save remote copy to S3
+        super(SilentFileStorage, self).save(name, content)
+        # Only save .scss and .js files locally
+        if ".scss" in name or ".js" in name:
+            # restore the clean file_content
+            content.file = file_content
```

Comment:
why do we do this ? would content.file change after we invoke L60 ?

Reply:
We have gzip enabled and from what I can tell, it's applied locally before the file is uploaded. We might not need it, but because we're saving multiple copies, I didn't want to risk losing the content.